wordMadness.js

```diff
@@ -0,0 +1,100 @@
+const wordMadnessWord = document.getElementById('word')
+const wordMadnessInput = document.getElementById('userInput')
+const wordMadnessGlossary = document.getElementById('glossary')
+const matchesInSetButton = document.getElementById('matches')
+const wrongCountInSetButton = document.getElementById('wrong')
+
+const data = [
```

Comment:
add a comment explaining why this is needed + rename to `mockData` 
+ add a `TODO: remove after real data is in place` to not forget